### DISCUSSION TITLE
row level security

### DISCUSSION THREAD

**Discussion by acivitillo** [nocodb/nocodb#640] [Features]
I am considering to use nocodb as an HRMS. In an HRMS we have employee information that should be visible only to the employee and their manager, this requires Row Level Security on top of the Project level one.

Are you guys considering RLS?

**Comment by o1lab** [comment]
RLS is there in nocodb but currently are tucked away ( its in API ACL - you can click on column name (header) and you should be able to set additional rules - but they won't work yet ) . To make them work with nested tables - we've to rethink a bit. Hence we havent documented them.

**Reply by MaxZhuravlev** [reply]
Very needed feature for typical business-processes.

RLS is one of the reasons, why "spreadsheets" users have to use separate tables with same structure for separate emploee's, and are eager of DB-solution where same data structure is being shared among lots of users without pain, but without security problems as well.

**Comment by remcohh** [comment]
To be honest, to me it is unbelievable that non of the airtable alternatives, including airtable itself implements row level security. This would be a winning feature in my opinion

**Comment by pemontto** [comment]
The way Supabase does this is very powerful, but also tightly coupled with Postgres:
https://supabase.com/docs/guides/auth/row-level-security

**Comment by mayeulk** [comment]
Discussing this at:
https://github.com/nocodb/nocodb/issues/6076
(unsure which is the best place to discuss this).
